### DISCUSSION TITLE
Why am I next config. js file to set the image to be unoptimized does not work?

### DISCUSSION THREAD

**Discussion by hexiaochen666** [vercel/next.js#38487] [Help] [answered]
Why am I next config. js file to set the image to be unoptimized does not work?

/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  images: {
    domains: ['assets.acme.com'],
  },
  experimental: {
    images: {
      unoptimized: true,
    },
  },
}

module.exports = nextConfig

**Comment by valcosmos** [comment, marked as answer]
Hello, maybe u can try this.

Write it in the Image tag, not in `next.config.js`.


```ts
import type { NextPage } from 'next'
import Head from 'next/head'
import Image from 'next/image'
import styles from '../styles/Home.module.css'

const Home: NextPage = () => {
  return (
    <div>
      <Image
        src="/sunset1.jpg"
        alt="Sunset"
        width={128}
        height={128}
        unoptimized={true}
        priority
        layout="fixed"
      />
    </div>
  )
}

export default Home
```

**Reply by hexiaochen666** [reply]
Thanks,it is useful.But why  is not work in next.config.js?

**Reply by valcosmos** [reply]
Hello, I have no idea why is not work in `next.config.js`. Maybe the property `unoptimized` belongs to the **experimental** configuration in `next.config.js`, there may be a problem.

**Reply by hexiaochen666** [reply]
ok  bro

**Reply by ycheng** [reply]
for nextjs 12.3.1, try
```
const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
  images: {
    unoptimized: true,
  },
}
```

in next.config.js.

**Reply by joehsn** [reply]
can anyone tell, why you don't want to optimize the images?
or `unoptimized` property has another effect on how the image will be displayed?

**Reply by SanjeebLama** [reply]
Hi Youssef,
Are you aware that by using next/image -> vercel will optimize your images and lets you optimize 1000 images in hobby plan and 5000 images in Pro plan after that you need to pay 5$ for every 1000 images.

My client don't want to pay extra, that's why I am disabling the optimization.

**Reply by joehsn** [reply]
Ooh,  I don't remember if I did read vercel plans before, but I thought it was a technical matter.
Anyway thank you for your help, appreciated. 🫰

**Reply by SanjeebLama** [reply]
Hey @RuttvikKheni,
After adding unoptimized in next config file.
Your images have stopped rendering because they are still being served as "next/image". To fix this, you can add the "unoptimized" keyword to all the next/image components.
Like:

`  <Image src={src} layout="fill" alt={alt} unoptimized />`

**Reply by aaronryden** [reply]
hi @SanjeebLama,

why is `unoptimized` on the <image /> component necessary?

why doesn't the global config setting take effect ?
```
 images: {
    unoptimized: true,
  },
```

**Reply by kumardeepam** [reply]
Image optimization cost is veru high, so in those cases its useful to keep the optimization off.
